### DISCUSSION TITLE
Online / offline indicator

### DISCUSSION THREAD

**Discussion by ghost** [rustdesk/rustdesk#1800] [Q&A] [answered]
Unless I'm missing something there is no way to distinguish between online and offline endpoints.

My suggestion - some way to easily distinguish between online and offline endpoints.

**Comment by pedjas** [comment]
There is actually third state: UNKNOWN, if you have some remotes listed but they are not at the same host as you.

**Comment by rustdesk** [comment, marked as answer]
https://github.com/rustdesk/rustdesk/discussions/2786
